config/routes.rb

```diff
@@ -20,7 +20,7 @@
 
   resources :workspaces, except: :new do
     member do
-      post 'duplicate'
```

Comment:
This is the only thing I'm scratching my head on. Why did this need to get replaced?

Reply:
Also noting that yes, workspace duplication definitely still works... so was this vestigial?